packages/common/src/directives/ng_optimized_image/ng_optimized_image.ts

```diff
@@ -448,6 +555,23 @@ function assertNotBase64Image(dir: NgOptimizedImage) {
   }
 }
 
+/**
+ * Verifies that the 'sizes' only includes responsive values.
+ */
+function assertNoComplexSizes(dir: NgOptimizedImage) {
+  let sizes = dir.sizes;
+  if (sizes) {
+    if (sizes.match(/((\)|,)\s|^)\d+px/)) {
```

Comment:
It looks like we can join 2 `if`s:
```suggestion
  if (sizes && sizes.match(/((\)|,)\s|^)\d+px/)) {
```

Reply:
done